src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -389,6 +389,38 @@ private static void close(final Closeable res) {
      * Stream monitor.
      */
     private static final class Monitor implements Callable<Void> {
+
+        /**
+         * Maximum number of log lines for a stack trace. Set to 1000
+         * to avoid storing too many lines in memory before flushing.
+         */
+        private static final int MAX_STACK_LENGTH = 1000;
+        /**
+         * Prefix "at ". Needed to check if line is part of stack trace.
+         */
+        private static final String PREFIX_AT = "at ";
```

Comment:
@dean-e-clark this looks like http://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html, can you inline all redundant variables?